addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -89,6 +89,7 @@ def _model_sorting_key(self, leave_type):
         ('half_day', 'Half Day'),
         ('hour', 'Hours')], default='day', string='Take Time Off in', required=True)
     unpaid = fields.Boolean('Is Unpaid', default=False)
+    is_public_holiday_include = fields.Boolean('Public Holiday Included', default=False, help="Public holidays should be counted in the leave duration when applying for leaves")
```

Comment:
I would rename this field to `include_public_holidays_in_duration`. I know it's a bit long, but I feel it more completely captures what this variable does

Reply:
Hello @robcoekaerts 

I have done the changes.thanks